[plugin-web-app] Unify syntax of step saving number of opened tabs (#3873)
vividus-framework/vividus: docs/modules/plugins/pages/plugin-web-app.adoc

When I save table to SCENARIO variable `table`
----

> ==== Save a number of opened tabs to the variable

Reviewer: ```suggestion
==== Save the number of opened tabs
```

> This step allows you to save a number of currently opened tabs in the browser.

Reviewer: ```suggestion
Saves the number of currently opened tabs in the browser to the variable.
```

[source,gherkin]
----
When I save number of open tabs to $scopes variable `$variableName`
----

* `$scopes` - xref:commons:variables.adoc#_scopes[The comma-separated set of the variables scopes].
* `$variableName` - The variable name to refer to the variable value.

> .Save the first table on the page into a variable named 'table'

Reviewer: need to be updated
